scripts/pi-hole/js/index.js

```diff
@@ -450,22 +454,19 @@ function updateForwardDestinationsPie() {
       values = [];
 
     // Collect values and colors
-    Object.keys(data.forward_destinations).forEach(function (key) {
-      var value = data.forward_destinations[key];
-
-      if (key.indexOf("|") !== -1) {
-        key = key.substr(0, key.indexOf("|"));
-      }
+    for (const key of Object.keys(data.forward_destinations)) {
+      let value = data.forward_destinations[key];
+      let altKey = key.indexOf("|") !== -1 ? key.substr(0, key.indexOf("|")) : key;
```

Comment:
needed to define another variable here because `key` is `const`, and therefore not changeable. 

Interestingly `xo:fix` suggested exactly the same for this conversion, except without the additional line. Of course that failed - I guess `fix` is not fool proof